content/800-data-platform/050-data-proxy/100-use-data-proxy.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: 'Use Data Proxy'
+metaTitle: 'Configure Data Proxy in a serverless application'
+metaDescription: 'Enable Data Proxy for your serverless application'
+tocDepth: 1
+---
+
+<TopBlock>
+
+To use the Data Proxy for your serverless application, you must first [create a project](/data-platform/projects/create) in Prisma Data Platform. When you create the project, you do the following:
```

Comment:
Why the mentions of  "serverless application" - how do I configure it for my non serverless app? (You do exactly the same - serverless is just one use case of deployment method of the Data Proxy)